### ISSUE TITLE
Error with octokit when submitting to winget package repo

### ISSUE THREAD

**Issue by aclinick** [microsoft/winget-create#158]
# Brief description of your issue

# Steps to reproduce

<!  wingetcreate new https://software.vc.logitech.com/downloads/tune/LogiTuneInstall.exe and answer default to all options then say yes to publish.  Logged into Github using my Microsoft accredited account aclinick and auth succeeded

# Expected behavior

# Environment

```none
Windows Package Manager Manifest Creator version? (wingetcreate.exe -?) 0.3.0.6

Any other software?
```

**Comment by ryfu-msft** [comment]
Hey @aclinick,

I am not able to repro this issue. Could you try again and let me know what the associated error message is when it failed to submit to the winget repo? Thanks!

**Comment by KevinLaMS** [comment]
@ryfu-msft Yeah, well I am.  :P 

![image](https://user-images.githubusercontent.com/2146880/159574930-7dc4a7f6-c9aa-492a-9bb8-08ca65563caa.png)

**Comment by ryfu-msft** [comment]
This issue is related to issue #287 and has been resolved by the changes from PR #289.
